app/Community/Requests/UpdateProfileRequest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Community\Requests;
+
+use App\Models\User;
+use Illuminate\Foundation\Http\FormRequest;
+
+class UpdateProfileRequest extends FormRequest
+{
+    public function authorize(): bool
+    {
+        /** @var User $user */
+        $user = $this->user();
+
+        if ($user->isBanned()) {
+            return false;
+        }
+
+        $isMottoBeingUpdated = $this->input('motto') !== $user->Motto;
```

Comment:
Does this need to check if a motto is being provided?

Reply:
Added in latest.